scripts/prepare_sector_network.py

```diff
@@ -2169,19 +2189,19 @@ def add_industry(n, costs):
         lifetime=costs.at['Fischer-Tropsch', 'lifetime']
     )
 
-    n.add("Load",
-        "naphtha for industry",
-        bus="EU oil",
+    n.madd("Load",
+        ["naphtha for industry"],
+        bus=vars(spatial)["oil"].nodes,
```

Comment:
```suggestion
    n.add("Load",
        "naphtha for industry",
        bus=spatial.oil.nodes[0],
```
I prefer `add` over `madd` here, because the component list must have only one item

Reply:
I would prefer `madd` since it is easier than to make the oil nodes also spatially resolved. One would have to add this only to ` define_spatial()` and add a switch in the `config.yaml`.

Reply:
Alright, then just:
```suggestion
    n.madd("Load",
        ["naphtha for industry"],
        bus=spatial.oil.nodes,
```